README.md

```diff
@@ -37,185 +41,198 @@ Simple parsing for node.js.
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-What is nearley?
-----------------
-nearley uses the Earley parsing algorithm augmented with Joop Leo's
-optimizations to parse complex data structures easily. nearley is über-fast and
-really powerful. It can parse literally anything you throw at it.
-
-nearley is used by [artificial
-intelligence](https://github.com/ChalmersGU-AI-course/shrdlite-course-project)
-and [computational
-linguistics](https://wiki.eecs.yorku.ca/course_archive/2014-15/W/6339/useful_handouts)
-classes at universities, as well as [file format
-parsers](https://github.com/raymond-h/node-dmi), [markup
-languages](https://github.com/bobbybee/uPresent) and [complete programming
-languages](https://github.com/bobbybee/carbon). It's an npm [staff
-pick](https://www.npmjs.com/package/npm-collection-staff-picks).
-
-Why do I care?
---------------
-
-nearley can parse what other JS parse engines cannot, because it uses a
-different algorithm. The Earley algorithm is *general*, which means it can
-handle *any* grammar you can define in BNF. In fact, the nearley syntax is
-written in *itself* (this is called bootstrapping).
+## Introduction
+
+nearley compiles grammar definitions from a simple syntax resembling [BNF](https://en.wikipedia.org/wiki/Backus–Naur_form) to a JS representation.
```

Comment:
Introduction should also describe why people like nearley: it's user-friendly, easy to get started, works on both the browser and node, etc.

I can take care of this change later though, don't worry about it.